assets/js/lazyload.js

```diff
@@ -79,27 +79,40 @@ function ebLazyLoadImages () {
   }
 }
 
+// Check if the document is ready for lazyloading
+function ebReadyForLazyloading () {
+  const readyForLazyloading = document.body.getAttribute('data-index-targets') &&
+                            document.body.getAttribute('data-ids-assigned')
+  if (readyForLazyloading) {
+    return true
+  } else {
+    return false
+  }
+}
+
 // Wait for data-index-targets to be loaded
 // and IDs to be assigned before lazyloading.
 // Otherwise intersectionObserver can miss images,
 // though I don't know why :-(
 function ebPrepareForLazyLoading () {
   'use strict'
 
-  const lazyImagesObserver = new MutationObserver(function (mutations) {
-    mutations.forEach(function (mutation) {
-      if (mutation.type === 'attributes') {
-        if ((settings.dynamicIndexing === false || document.body.getAttribute('data-index-targets')) &&
-                        document.body.getAttribute('data-ids-assigned')) {
+  if (ebReadyForLazyloading()) {
```

Comment:
@arthurattwell this is very nitpicky, but can please use one of either 'LazyLoading' or 'Lazyloading'? We've got the latter here in this conditional, and the former in the name of the parent function

Reply:
Fixed in [5b6b881](https://github.com/electricbookworks/electric-book/pull/737/commits/5b6b8810bc683a7f56ffbb720181454bdcd4c7a6). We've used 'lazyLoad' over 'lazyload' elsewhere, so going with that.